cypress/utils/constants.js

```diff
@@ -4,49 +4,7 @@
  */
 
 export const BASE_PATH = Cypress.config('baseUrl');
+export const BACKEND_BASE = Cypress.env('openSearchUrl');
 
-/**
- *****************************
- AD PLUGIN CONSTANTS
- *****************************
- */
-
-export const AD_FIXTURE_BASE_PATH = 'plugins/anomaly-detection-dashboards-plugin/'
-
-export const BASE_AD_PATH = BASE_PATH + '/app/anomaly-detection-dashboards#';
-
-export const BASE_AD_DASHBOARDS_PATH = BASE_AD_PATH + '/dashboard';
-
-export const BASE_AD_DETECTOR_LIST_PATH = BASE_AD_PATH + '/detectors';
-
-export const BASE_AD_CREATE_AD_PATH = BASE_AD_PATH + '/create-ad';
-
-export const BASE_AD_NODE_API_PATH = BASE_PATH + '/api/anomaly_detectors';
-
-export const AD_GET_DETECTORS_NODE_API_PATH =
-  BASE_AD_NODE_API_PATH + '/detectors*';
-
-export const AD_GET_INDICES_NODE_API_PATH =
-  BASE_AD_NODE_API_PATH + '/_indices*';
-
-export const AD_GET_MAPPINGS_NODE_API_PATH =
-  BASE_AD_NODE_API_PATH + '/_mappings*';
-
-export function getADStartDetectorNodeApiPath(detectorId) {
-  return BASE_AD_NODE_API_PATH + '/detectors/' + detectorId + '/start';
-}
-
-export function getADStopDetectorNodeApiPath(detectorId) {
-  return BASE_AD_NODE_API_PATH + '/detectors/' + detectorId + '/stop';
-}
-
-export const TEST_DETECTOR_ID = 'ulgqpXEBqtadYz9j2MHG';
-
-// TODO: when repo is onboarded to typescript we can convert these detector states into an enum
-export const DETECTOR_STATE_DISABLED = 'Stopped';
-
-export const DETECTOR_STATE_INIT = 'Initializing';
-
-export const DETECTOR_STATE_RUNNING = 'Running';
-
-export const DETECTOR_STATE_FEATURE_REQUIRED = 'Feature required';
+export * from './plugins/anomaly-detection-dashboards-plugin/constants';
+export * from './plugins/index-management-dashboards-plugin/constants';
```

Comment:
Any specific reason for exporting here? Seems we could just export and import at the plugin-dir level to simplify things.

Reply:
Through this way, tests look cleaner to me, all constants import from single place. 
I don't think import contants from plugin constant file is what we want, we still want a single source, but just refactor those to not explode the main constants.js.

Reply:
It is very common to have constants only scoped for their specific modules / components within a project - this seems to add unnecessary overhead of importing a constant from a central constants file which is exporting it from some local place to begin with. Additionally, it adds some confusing loading issues as mentioned [here](https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/94#discussion_r810173438), which seems like a common use case, where global constants may want to be used by local / plugin-level constants. 

The tests would still look clean, as they can import from `./constants` or `./utils/constants`, instead of `../../constants`

@tianleh thoughts on this?

Reply:
I would prefer each test file importing their own constant file. The `cypress/utils/constants.js` ideally shall only contain common ones.

Reply:
After checking further with @bowenlan-amzn , it is because ISM tests need both the common constants and ISM specific ones. Thus they will need to always write two lines of import in their test files which is indeed annoying. Thus doing in current way could just use one line of import. I am ok with this.